.github/workflows/ci.yml

```diff
@@ -1234,24 +1239,26 @@ jobs:
     timeout-minutes: 3
     permissions:
       checks: read
-    needs:  # BELOW LIST IS IN ALPHABETICAL ORDER
+    needs: # BELOW LIST IS IN ALPHABETICAL ORDER
       - actionlint
       - build-operate-backend
       - build-platform-frontend
       - commitlint
-      - consolidated-unit-tests
+      - operate-backend-unit-tests
+      - optimize-backend-unit-tests
+      - tasklist-backend-unit-tests
+      - zeebe-unit-tests
```

Comment:
List should be alphabetical to make merge conflict resolution and completeness checks easier, please reorder appropriately.